lm/lm_image.py

```diff
@@ -0,0 +1,120 @@
+__all__ = ["LmImageJob"]
+
+import shutil
+
+from sisyphus import *
+from i6_core import rasr
+from i6_core import util
+
+Path = setup_path(__package__)
+
+
+class LmImageJob(rasr.RasrCommand, Job):
+    """
+    pre-compute LM image without generating global cache
+    """
+
+    def __init__(
+        self,
+        crp,
+        extra_config=None,
+        extra_post_config=None,
+        encoding="utf-8",
+        renormalize=False,
+    ):
+        kwargs = locals()
+        del kwargs["self"]
+
+        self.text_file = text_file
+        self.renormalize = renormalize
+
+        self.config, self.post_config, self.num_images = LmImageJob.create_config(
+            **kwargs
+        )
+        self.exe = self.select_exe(crp.lm_util_exe, "lm-util")
+
+        self.log_file = self.log_file_output_path("lm_image", crp, False)
+        self.lm_images = {
```

Comment:
if this is an output of the Job, maybe we want to prefix the member with "out_"?
```suggestion
        self.out_lm_images = {
```